lib/src/formatter.dart

```diff
@@ -50,11 +50,12 @@ class LcovFormatter implements Formatter {
       }
 
       buf.write('SF:$source\n');
-      v.keys.toList()
-        ..sort()
-        ..forEach((int k) {
-          buf.write('DA:$k,${v[k]}\n');
-        });
+      final lines = v.keys.toList()..sort();
+      lines.forEach((int k) {
+        buf.write('DA:$k,${v[k]}\n');
+      });
```

Comment:
Use a for-loop here rather than `.forEach()`

https://www.dartlang.org/guides/language/effective-dart/usage#avoid-using-iterableforeach-with-a-function-literal

Reply:
Actually never mind, I'll follow up and eliminate my previous shameful commit separately ;)